supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -98,3 +98,132 @@ def to_json(self) -> Dict[str, Any]:
             "users": users_json,
             "nextPaginationToken": self.next_pagination_token,
         }
+
+
+class DashboardUsersGetResponseWithMetadata(APIResponse):
+    status: str = "OK"
+
+    def __init__(
+        self, users: List[Dict[str, Any]], next_pagination_token: Optional[str]
```

Comment:
`List[Dict[str, Any]]` does not enforce the type of the object right? Doesnt this defeat the purpose of strongly typing the APIs?

Reply:
fixed.